### DISCUSSION TITLE
SyntaxError: Unexpected token 'export'

### DISCUSSION THREAD

**Discussion by vaynevayne** [vercel/next.js#53830] [Help]
### Summary

My component library docs project is not working properly,
https://github.com/vaynevayne/get-ui 

```jsx
pnpm run dev:docs
```

<img width="1048" alt="image" src="https://github.com/vercel/next.js/assets/124666577/df3d6ccb-ce86-4d47-94a8-d6b91d454b98">

### Example

https://github.com/vaynevayne/get-ui

**Comment by devjiwonchoi** [comment]
Looks fine to me. Did you manage to fix it?

**Reply by vaynevayne** [reply]
yes  I fixed  it。
